accname/index.html

```diff
@@ -706,8 +706,8 @@ <h4>Computation steps</h4>
                   </div>
                 </li>
                 <li id="comp_text_node">
-                  <span id="step2G"><!-- Don't link to this legacy numbered ID. --></span><em>Text Node:</em> Otherwise, if the <code>current node</code> is a Text [=Node=], return its textual
-                  contents.
+                  <span id="step2G"><!-- Don't link to this legacy numbered ID. --></span><em>Text Node:</em> Otherwise, if the <code>current node</code> is a [=exclusive Text node|text node=], return
```

Comment:
```suggestion
                  <span id="step2G"><!-- Don't link to this legacy numbered ID. --></span><em>Text Node:</em> Otherwise, if the <code>current node</code> is an [=exclusive Text node|text node=], return
```

Reply:
This would result in “Otherwise, if the current node is an [sic] text node, return”, right?

The [preview deployment](https://deploy-preview-2274--wai-aria.netlify.app/accname/#comp_text_node) has this:
> G. Text Node: Otherwise, if the current node is a [text node](https://dom.spec.whatwg.org/#exclusive-text-node), return its textual contents.

Reply:
ah - sorry got it the wrong way round. My mistake.